production/mempool-config.testnet.json

```diff
@@ -22,7 +22,8 @@
   "CORE_RPC": {
     "PORT": 18332,
     "USERNAME": "__BITCOIN_RPC_USER__",
-    "PASSWORD": "__BITCOIN_RPC_PASS__"
+    "PASSWORD": "__BITCOIN_RPC_PASS__",
+    "SSL":false
```

Comment:
You can revert all production config additions, as we don't need to add when the default config is the same. Keeps them cleaner.